clc/paths/paths_test.go

```diff
@@ -0,0 +1,78 @@
+package paths_test
```

Comment:
Tests for `ResolveLogPath`, `ResolveConfigPath`, `Schemas` are missing, maybe we can test them too to increase coverage. Ignore this if you think it is unnecessary.

Reply:
Added more tests at: https://github.com/hazelcast/hazelcast-commandline-client/pull/160/commits/8b93697886f93b70a5932dda90559e8d4d075f35